usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -1,7 +1,10 @@
 
-if grep -q '^multipath' "$LAYOUT_FILE" ; then
+# Activating multipath if BOOT_OVER_SAN variable is true
+# or if multipath device are present in LAYOUT_FILE
+if grep -q '^multipath' "$LAYOUT_FILE" || is_true "$BOOT_OVER_SAN" ; then
     Log "Activating multipath"
     modprobe dm-multipath >&2
+    [ ! -f /etc/multipath.conf ] && touch /etc/multipath.conf
```

Comment:
I assume that "afterwards the
TARGET_FS_ROOT/etc/multipath.conf must be created"
happens via the backup restore?